src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const perDay = 40;
+  let total = days * perDay;
+
+  if (days >= 7) {
+    total = total - 50;
+  } else if (days >= 3) {
+    total = total - 20;
+  }
+
+  return total;
```

Comment:
The function name 'calculateRentalCost' is descriptive and clear, which is good. However, the function uses hard-coded literals for the discount values and the minimum number of days for discounts, which should be avoided according to the checklist.